instrumentation/aws_sdk/test/opentelemetry/instrumentation_test.rb

```diff
@@ -54,7 +54,7 @@
         _(last_span.attributes['rpc.service']).must_equal 'SNS'
         _(last_span.attributes['rpc.method']).must_equal 'Publish'
         _(last_span.attributes['aws.region']).must_equal 'us-stubbed-1'
-        _(last_span.attributes['db.system']).must_equal nil
+        assert_nil last_span.attributes['db.system']
```

Comment:
💁🏼 I think what we want here is [`must_be_nil`](https://docs.seattlerb.org/minitest/Minitest/Expectations.html#method-i-must_be_nil)
```suggestion
       _(last_span.attributes['db.system']).must_be_nil
```